### PR TITLE
Interface for fillna. SQL portion will be separate.

### PR DESCRIPTION
**Pull request by kvnkho**

### CODE REVIEW

File: fugue/workflow/workflow.py [hunk 1]
```diff
@@ -531,13 +532,35 @@ def distinct(self: TDF) -> TDF:
         df = self.workflow.process(self, using=Distinct)
         return self._to_self_type(df)
 
-    def dropna(self: TDF, **kwargs: Any) -> TDF:
+    def dropna(
```
Comment (WangCHX) [inline fugue/workflow/workflow.py:535]: Nice.

**Review by WangCHX — APPROVED** [review]
good stuff. LGTM. need to wait for @goodwanghan 's approval.

File: fugue/execution/execution_engine.py [hunk 2]
```diff
@@ -368,6 +368,26 @@ def dropna(
         """
         pass
 
+    @abstractmethod
+    def fillna(
+        self, df: DataFrame, value: Any, metadata: Any = None, subset: List[str] = None
```
Comment (goodwanghan) [inline fugue/execution/execution_engine.py:373]: It's better to move dropna and fillna metadata to the last parameter

**Comment by goodwanghan** [comment]
Looks perfect. Thanks Kevin. Great work.

**Review by goodwanghan — APPROVED** [review]

**Pull request merged**